src/main/java/org/fenixedu/academic/domain/accounting/Event.java

```diff
@@ -717,18 +814,20 @@ protected YearMonthDay calculateNextEndDate(final YearMonthDay yearMonthDay) {
     }
 
     public Money getReimbursableAmount() {
-        if (!isClosed() || !hasEventCloseDate()) {
-            return Money.ZERO;
-        }
+//        if (!isClosed() || !hasEventCloseDate()) {
```

Comment:
is it necessary to keep this commented code for future changes or is the calculatePayedAmountByPerson() method call the correct implementation already?